.github/workflows/npm-publish.yml

```diff
@@ -19,31 +19,30 @@ jobs:
         with:
           ref: ${{ github.event.inputs.tag }}
 
-      - name: Match semver pattern
-        uses: actions-ecosystem/action-regex-match@v2
-        id: regex-match
-        with:
-          text: ${{ github.event.inputs.tag }}
-          regex: '^v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$'
-
       - name: Check package version
         id: cpv
         uses: PostHog/check-package-version@v2
 
+      - run: npm install semver
```

Comment:
i think this can just be `npx semver $version`, and it'll fail if it's not valid? (and the proper version number will be echoed, so that output could be used in the next step)

Reply:
Updated to use `npx semver ${{ inputs.tag }}`. Since this fails if not valid, removed few lines from a later step that were doing that explicitly.